Task3/UI/style.css

```diff
@@ -0,0 +1,220 @@
+body{
+    margin: 0;
+    padding: 0;
+}
+
+.divHat{
+    position:fixed;
+    z-index:2;
+    width:90%;
+    top:0;
+    padding-left: 10%;
+    background-color:white;
+    border-bottom: solid 1px rgba(0,0,0,0.3);
+}
+
+.header{
+    margin-right:7vw;
+    margin-top:0;
+    margin-bottom:0;
+    padding:0;
+    font-family: 'Pacifico', cursive;
+    font-size: 2.5vw;
+    float:left;
+}
+
+.buttonHead{
+    margin-top:1.3vh;
+    margin-right:7vw;
+    font-size: 2vw;
+    font-family: 'Patrick Hand', cursive;
+    color:rgba(0,0,0,0.3);
+    background-color: white;
+    border: 2px solid;
+    border-radius:10px ;
+    border-color: rgba(0,0,0,0.3);
+    transition-duration: 0.5s;
+    float:left;
+}
+.buttonHead:hover{
+    background-color: rgba(0,0,0,0.13);
+    color: white;
+}
+
+.fa-image{
+    float:left;
+    font-size:6vh;
+    margin-top:1vw;
+    margin-right:0.5vw;
+}
+.nameHead{
+    margin-top:2.5vh;
+    margin-right:0.5vw;
+    float:left;
+    font-family: 'Rancho', cursive;
+    font-size:2vw;
+}
+
+.divFilter{
+    display: none;
+    position:fixed;
+    z-index:3;
+    background-color:white;
+    left:27%;
+    top:11.5vh;
+    width: 20%;
+    margin: 1.5%;
+    border: solid 3px rgba(0,0,0,0.5);
+    border-radius: 5px;
+}
+
+.divFilterElement{
+    border-bottom: solid 3px rgba(0,0,0,0.5);
+    padding: 5px;
+}
+
+.filterText{
+    border: 1px solid rgba(0,0,0,0.5);
+    font-size: 3vh;
+    font-family: 'Open Sans Condensed', sans-serif;
+    width: 13vw;
+    display: block;
+    box-sizing: border-box;
+    padding: 0.3vh;
+    border-radius: 60px;
+    color: black;
+    font-weight: 100;
+    letter-spacing: 0.01em;
+    float:left;
+    margin-right:1vw;
+}
+
+.checkbox{
+    width: 2.5vw;
+    height: 5vh;
+    display: block;
+    margin-top: 1px;
+    margin-bottom: 1px;
+}
+
+.divButtonFilter{
+    padding: 3px;
+    text-align:center;
+}
+
+.buttonSubmit{
+    border : solid 2px rgba(0,0,0,0.5);
+    border-radius : 5px;
+    font-size : 1.5vw;
+    color : #ffffff;
```

Comment:
please, choose one type of color representation
https://www.w3schools.com/cssref/css_colors_legal.asp